.github/workflows/create-release-branch.yaml

```diff
@@ -0,0 +1,28 @@
+name: Create release branch
+
+on:
+  workflow_dispatch:
+    inputs:
+      newVersionNumber:
+        description: 'New version number'
+        required: true
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: fregante/setup-git-user@v1
+
+      - name: Replace version
+        run: |
+          sed -i -e '/version =/ s/= .*/= ${{ github.event.inputs.newVersionNumber }}/' ./plugins/sqlfluff-templater-dbt/setup.cfg
+          sed -i -e '/version =/ s/= .*/= ${{ github.event.inputs.newVersionNumber }}/' ./setup.cfg
+          sed -i -e '/stable_version =/ s/= .*/= ${{ github.event.inputs.newVersionNumber }}/' ./setup.cfg
```

Comment:
We could make the version number optional and if a version number is NOT provided, then it auto bumps to the next minor version. If you want a major version then you have supply it.

But probably best to just make it mandatory for simplicity.

Reply:
Looking at recent history, we do lots of patch versions (0.13.1 -> 0.13.2) and lots of minor versions (0.13.1 -> 0.14.0). You would want it to default to doing minor versioning?

Yeah it would add a little complexity to this because we have to read, parse, and correctly bump the current version instead of just blasting it away. I think it could be a separate PR.

Reply:
Yeah I think leave as is. Was just musing...

Resolving.